cypress/integration/visitorCanAuthorizeUsingOAuth.js

```diff
@@ -0,0 +1,14 @@
+describe("User can log in", () => {
+
+  it("Clicking on 'Log in with Facebook' opens dialog window", () => {
+    cy.visit("http://localhost:3001", {
+      onBeforeLoad(win) {
+        cy.stub(win, 'open')
+      }
+    });
+    cy.get("#login-button").click();
+    cy.get(".button .facebook").click()
+    cy.window().its('open').should('be.called')
+  });
+
```

Comment:
shouldn't we have an assertion to test when the facebook auth goes through? and a sad path when it doesn't?

Reply:
Check the link in the README